### PR TITLE
close #16 for every one with high res screen

### LINKED ISSUE #16: use flex to reflow horizontally for wider screens

### PR DESCRIPTION
**Pull request by butlerx**
close #16

### CODE REVIEW

File: 2015/index.html [hunk 1]
```diff
@@ -35,20 +35,22 @@
...
-            <a href="../2016">Feb 2016</a>
-          </li>
-          <li>
-            <a href="..">Oct 2016</a>
-          </li>
-        </ul>
-      </header>
-      <div class="container playlist">
-        <iframe id="live" src="https://www.youtube.com/embed/videoseries?list=PLPaLJSHTDMFgYNnnEOjuvdm6WqOFTSQ9z" width="100%" height="400" marginwidth="0" marginheight="0" scrolling="no" frameborder="0" webkitallowfullscreen="" allowfullscreen=""></iframe>
+      <div class="main">
```
Comment (VoyTechnology) [inline 2015/index.html]: We could use an actual `<main>` HTML5 element

Comment (butlerx) [inline 2015/index.html]: This is more a case of a bad class name

Comment (VoyTechnology) [inline 2015/index.html]: It can be made better with `<main>` :)

**Comment by SeanHealy33** [comment]
Link issue in description

File: 2015/index.html (hunk 1 above)
Comment (butlerx) [inline 2015/index.html]: tbh when using it for appling style using class is better

**Comment by butlerx** [comment]
@SeanHealy33 what?

File: index.html [hunk 2]
```diff
@@ -37,33 +37,35 @@
...
-      </header>
-      <!-- Countdown is show up to 10 minutes before the event -->
-      <div class="countdown">
-        TechWeek Starts in:
-        <div class="countdown__counter">
-          <span>&nbsp;</span>
-          <span>&nbsp;</span>
-          <span>&nbsp;</span>
-          <span>&nbsp;</span>
+      <div class="main">
```
Comment (SeanHealy33) [inline index.html]: What voy said before ⚡️

File: 2015/index.html (hunk 1 above)
Comment (butlerx) [inline 2015/index.html]: this isnt the main content of the page the main content would be the video and evens cards

**Pull request merged**